tests/integration/api_v3/http/posStakes.test.js

```diff
@@ -21,59 +21,54 @@ const baseUrlV3 = `${baseUrl}/api/v3`;
 
 const {
 	badRequestSchema,
-	metaSchema,
 } = require('../../../schemas/httpGenerics.schema');
 
 const {
-	voteSchema,
-} = require('../../../schemas/api_v3/vote.schema');
+	stakeResponseSchema,
+} = require('../../../schemas/api_v3/stake.schema');
 
-const endpoint = `${baseUrlV3}/dpos/votes/sent`;
+const endpoint = `${baseUrlV3}/pos/stakes`;
 
-describe('Votes Sent (Votes) API', () => {
-	let refDelegate;
+describe('Stakes API', () => {
+	let refValidator;
 	beforeAll(async () => {
 		let response;
 		do {
 			// eslint-disable-next-line no-await-in-loop
 			response = await api.get(`${baseUrlV3}/dpos/delegates?limit=1`);
 		} while (!response.data);
-		[refDelegate] = response.data;
+		[refValidator] = response.data;
 	});
 
 	describe(`GET ${endpoint}`, () => {
 		it('Returns list of votes when requested for existing account by address', async () => {
-			const response = await api.get(`${endpoint}?address=${refDelegate.address}`);
-			expect(response.data).toMap(voteSchema);
+			const response = await api.get(`${endpoint}?address=${refValidator.address}`);
+			expect(response).toMap(stakeResponseSchema);
 			expect(response.data.votes.length).toBeGreaterThanOrEqual(1);
 			expect(response.data.votes.length).toBeLessThanOrEqual(10);
-			expect(response.meta).toMap(metaSchema);
-		});
-
-		it('Returns list of votes when requested for existing account by name', async () => {
-			if (refDelegate.name) {
-				const response = await api.get(`${endpoint}?name=${refDelegate.name}`);
-				expect(response.data).toMap(voteSchema);
-				expect(response.data.votes.length).toBeGreaterThanOrEqual(1);
-				expect(response.data.votes.length).toBeLessThanOrEqual(10);
-				expect(response.meta).toMap(metaSchema);
-			}
 		});
 
 		it('Returns list of votes when requested for existing account by address and limit=5', async () => {
-			const response = await api.get(`${endpoint}?address=${refDelegate.address}&limit=5`);
-			expect(response.data).toMap(voteSchema);
+			const response = await api.get(`${endpoint}?address=${refValidator.address}&limit=5`);
+			expect(response).toMap(stakeResponseSchema);
 			expect(response.data.votes.length).toBeGreaterThanOrEqual(1);
 			expect(response.data.votes.length).toBeLessThanOrEqual(5);
-			expect(response.meta).toMap(metaSchema);
 		});
 
 		it('Returns list of votes when requested for existing account by address, limit=5 and offset=1', async () => {
```

Comment:
Please update the test description at all places
```suggestion
		it('Returns list of stakes when requested for existing account by address, limit=5 and offset=1', async () => {
```